packages/venia-ui/lib/components/CartPage/ProductListing/productListingFragments.js

```diff
@@ -9,6 +9,8 @@ export const ProductListingFragment = gql`
                 id
                 name
                 sku
+                url_key
+                url_suffix
```

Comment:
Any specific reason why you're modifying the cart page's product list fragment in this PR? I would assume that would be done as part of #2557

Reply:
That's because when a product is added to the cart, the mini-cart won't update if those 2 keys are missed in the `productListingFragment`. `productListingFragment` is fetched in return for the add to cart mutation as part of the 
`CartPageFragment` of `cartPageFragments.gql.js`.

We could have added the Mini cart fragment as a requested response as well, but we have decided against it previously in another PR. That is also the reason why we have added `thumbnail { url }` as part of the `productListingFragment` previously (because the mini cart needs it to be updated).

Reply:
> That's because when a product is added to the cart, the mini-cart won't update if those 2 keys are missed in the productListingFragment.

http://develop.pwa-venia.com works without this addition. I'm able to add an item from the PDP and see it in the mini cart. I can add more items and still see them update in the mini cart. I can also increase/decrement the quantity of cart items within the /cart page and still see the mini cart update.

Regardless, my point is that you are already adding it in #2557 in order to link from the PDP, so you don't need to do this here, right? It's just scope creeping (in a minor way). Not asking you to remove it necessarily.

Reply:
At first, the mini cart was not updating if I did not have these keys as the return value, which is understandable. Hence I added them as part of this PR. They also happen to be needed as part of #2557 which was a parallel PR I was working on. These 2 happened un-related to each other. 

But now I think we don't need them anymore since 8 days ago, your PR #2505 was merged which requests the `MiniCartFragment` on the `addConfigurableProductToCart`. Since I never modified that file, it did not create a merge conflict. Nice catch dude. Removing them in this commit 53c89966135862476d8945bb4c8f838e02c2aab7.

You will understand why I had those 2 keys initially if you remove the `MiniCartFragment` and those 2 keys from the `productListingFragment` and try to add an item to the cart.